README.md

```diff
@@ -257,6 +262,10 @@ It has the same parameters as a normal rule:
 - **countAuthor**: If the pull request author should be considered as an approval.
 	- If the author belongs to the list of approved users (either by team or by users) his approval will be counted (requiring one less approvals in total).
 	- ** Optional**: Defaults to `false`
+- **excludeAuthor**: If the author belong to one of the teams and/or users in the list, the rule should be skipped.
```

Comment:
```suggestion
- ** allowedToSkipRule**: If the author belong to one of the teams and/or users in the list, the rule should be skipped.
```

Reply:
Good catch. I wrote it wrong (missed the last s) so the replace feature didn't get it